### DISCUSSION TITLE
Restore target application before injection when using Forms

### DISCUSSION THREAD

**Discussion by Gadgetroch** [espanso/espanso#1240] [Feature requests and Ideas]
Hi,

When i launch form with espanso, and i need to copy something, my cursor location change.

So my previous position, when espanso was launched is lost.

Can we make something for remember the cursor position when espanso form is launched ?

Thank's !

**Comment by federico-terzi** [comment]
Hey @Gadgetroch,

Thank you for the feedback! 

> and i need to copy something, my cursor location change.

Are you copying it from a different application or from the same one in which you triggered the expansion?

Cheers :)

**Comment by Gadgetroch** [comment]
From a different application.

Like if i need get some password or e-mail adresses

Le sam. 13 nov. 2021 à 14:32, Federico Terzi ***@***.***> a
écrit :

> Hey @Gadgetroch <https://github.com/Gadgetroch>,
>
> Thank you for the feedback!
>
> and i need to copy something, my cursor location change.
>
> Are you copying it from a different application or from the same one in
> which you triggered the expansion?
>
> Cheers :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/federico-terzi/espanso/issues/865#issuecomment-968069277>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALAM7ZUA4JECCUGSTGMTLCDULZSG5ANCNFSM5H5X2X3Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 

Cordialement,

Sette Rocco

**Comment by federico-terzi** [comment]
I see, thanks! I agree this would be useful. It's definitely not trivial to implement though, so I can't give you an ETA :)
